openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -86,6 +86,8 @@ class Meta(object):
     announcement = DateTimeField(null=True)
 
     # URLs
+    # Not allowing null per django convention; not sure why many TextFields in this model do allow null
+    banner_image_url = TextField()
```

Comment:
Thank you for this platform enhancement.
Where may I find documentation on how a "banner image" differs from a "course image"?

Reply:
> Where may I find documentation on how a "banner image" differs from a "course image"?

@nasthagiri I see some context here, but not sure that it qualifies as "documentation" : https://github.com/edx/edx-platform/blob/7afee25ce2150443eefe0ae60b81f3e4f6e61900/common/lib/xmodule/xmodule/course_module.py#L589-L609

Reply:
From a consumer perspective, it would be useful to know when the difference between the 2 images. Perhaps it would help to describe where each image is displayed?

Reply:
ping @gabor-boros - can you answer that question ^ ?

Reply:
@nasthagiri & @bradenmacdonald  Apologies for replying that late, I probably missed the notification about this comment. As far as I can see, the `banner_image` is [part of](https://github.com/edx/edx-platform/blob/9caf1e51e99d9f2cc6d1d649d4861ef11b3f3df1/cms/templates/settings.html#L449-L515) the `extended_course_details`. The original addition was part of https://github.com/edx/edx-platform/commit/f7f281b65b2617b3b5bf599dcc18e67ddf89acb0 as far as I can see, but I cannot find more information about that either.

From consumer perspective - in my mind - the difference is that the course image is kind of a thumbnail of a course while the banner image is like a cover image. So something like this:

![Screenshot 2020-12-08 at 17 24 17](https://user-images.githubusercontent.com/19173947/101510428-369d3d80-397a-11eb-909f-d1bd50d73832.png)